src/units.cpp

```diff
@@ -118,6 +121,17 @@ struct Units::UnitsImpl
     std::vector<Unit> mUnits; /**< A vector of unit defined for this Units.*/
 
     std::vector<Unit>::iterator findUnit(const std::string &reference);
+    /**
+     * @brief Returns the log of the scaling factor between two similar units
+     * 
+     *  The factor returned can be interpreted as u1 = (factor)*u2
```

Comment:
Hmm... I would consider `u1` to be our reference, so I would expect the returned factor to be interpreted as `u2 = factor*u1`.